AWSCore.podspec

```diff
@@ -12,7 +12,7 @@ Pod::Spec.new do |s|
   s.platform     = :ios, '8.0'
   s.source       = { :git => 'https://github.com/aws/aws-sdk-ios.git',
                      :tag => s.version}
-  s.frameworks   = 'UIKit', 'Foundation', 'SystemConfiguration'
+  s.frameworks   = 'CoreGraphics', 'UIKit', 'Foundation', 'SystemConfiguration', 'Security'
```

Comment:
Could you please explain why do we need CoreGraphics in here?

Reply:
There is some (very old) code in `AWSDDTTYLogger.m` that calls some CoreGraphics methods. UIKit includes the declarations, but not the definitions, so we need to link CoreGraphics as well.

https://github.com/aws/aws-sdk-ios/blob/a50e487cc71fc9345bdd55b8bdfe0eb08c7250ac/AWSCore/Logging/AWSDDTTYLogger.m#L696

Reply:
If you'd like to try this out for yourself, in this project, turn off 'Enable Modules' build setting and try to run the unit tests. You'll need to add all these frameworks in the link build phase.

Reply:
Is there anything else I can do to get this over the line?

Reply:
Hi, friendly bump here